droopler.install

```diff
@@ -2256,3 +2256,17 @@ function droopler_update_8138(&$sandbox) {
   \Drupal::service('d_update')->checklist()
     ->saveProgress(['v3.1' => $values]);
 }
+
+/**
+ * Droopler 3.1 Enabling d_mautic and updating d_content.
```

Comment:
This module is optional, so we don't have to install it by default with Droopler - please move the code to the Mautic module, to the `d_mautic_install()` function. The config change should happen only when we install d_mautic module on existing Droopler sites.

Reply:
@grzegorz-pietrzak-droptica fixed :)